app/build.gradle

```diff
@@ -29,7 +29,8 @@ androidExtensions {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation project(':domain')
-    implementation"org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+    implementation project(':data')
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
 
     implementation 'com.android.support:appcompat-v7:27.1.1'
```

Comment:
I know not all of these are you, but can you add variables for the version numbers instead of writing them out as strings